packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -100,7 +100,7 @@ export function isExplicitImportRequired(url: string): boolean {
   return !isJSRequest(url) && !isCSSRequest(url)
 }
 
-export function normalizeResolvedIdToUrl(
+function normalizeResolvedIdToUrl(
```

Comment:
Didn't we export this from `vite` entry point? 🤔 I think that was the idea

Reply:
It might be useful, but I don't see it exported from `packages/vite/src/node`.

Reply:
I remember I wanted to have it be exported for the browser mode, right now we just copy paste this function, but it's out of scope of this PR